src/etc/inc/gwlb.inc

```diff
@@ -33,15 +33,15 @@
 function return_dpinger_defaults()
```

Comment:
keep these... only leading tabs please (^\t)

Reply:
That violates PSR-2, which says spaces should be used throughout. (And even most tab-intended code bases use spaces for aligning comments.)

Reply:
Ok, I will revert them for now.

I really hope the PHP files will be PSR-2 compliant sooner than later, though. I like the code base to be clean and consistent.